.github/actions/composite/setupNode/action.yml

```diff
@@ -12,6 +12,10 @@ runs:
           package-lock.json
           desktop/package-lock.json
 
+    - name: Setup Mapbox SDK
```

Comment:
Using setupNode for this is fine imo since doing this is a prerequisite to run `npm install`. But if I understand correctly, you really need to run `./scripts/setup-mapbox-sdk` _before_ running `npm install` right? If that's the case you'll need to move this step to make it happen before the `setup-node` action above, because `setup-node` will automatically run `npm ci`.

Reply:
I think `npm ci` runs [here](https://github.com/Expensify/App/blob/dc833c22c4c8141375f384a77c9e06e076113c00/.github/actions/composite/setupNode/action.yml#L33) if `node_modules` is not cashed.

Looking at the example code in the [documentation](https://github.com/actions/setup-node#:~:text=version%3A%2018-,%2D%20run%3A%20npm%20ci,-%2D%20run%3A) of setup-node GH action, they're running `npm ci` separately after `uses: actions/setup-node@v3` so `setup-node` probably just installed `node` itself but don't install npm dependencies.